src/serac/physics/heat_transfer.hpp

```diff
@@ -426,8 +412,25 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
   void setMaterial(DependsOn<active_parameters...>, MaterialType material)
   {
     ThermalMaterialIntegrand<MaterialType> integrand(material);
-    residual_->AddDomainIntegral(Dimension<dim>{}, DependsOn<0, 1, 2, active_parameters + NUM_STATE_VARS...>{},
-                                 integrand, mesh_);
+
+    // This does not compile
+    residual_->AddDomainIntegral(Dimension<dim>{}, DependsOn<0, 1, NUM_STATE_VARS + active_parameters...>{}, integrand,
+                                 mesh_);
+
+    // This does compile
```

Comment:
@samuelpmishLLNL , this is where the problem is. I can compile this using the commented out generic lambda, but not using the integrand struct. The compiler says the appropriate paren operator cannot be found.

Reply:
Thanks for the tip, I'm pulling this branch now to take a look

Reply:
looks like it was just missing a `const` on the paren operator (since the functor being invoked was marked `const`, it wasn't allowed to use the non-const member function).

On my mac, all tests are passing except for `boundary_cond` when run in parallel (it passes when run without mpirun).

Reply:
Ahhh that makes sense. Thank you for figuring this out!